assets/jump-dev-workshops/2024/schedule.toml

```diff
@@ -32,7 +32,7 @@ url = "https://youtu.be/JhgHKYvga2s"
 [talks.1_09_45]
 speaker = "Joaquim Dias Garcia"
 title = "Differentiating Parametric JuMP Models"
-# slides = ""
+slides = "diasgarcia.pdf"
```

Comment:
So you need to run `julia assets/jump-dev-workshops build_schedule.jl 2024` to actually update the HTML :smile: